src/components/SortDropdown.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { useState } from "react";
+
+
+export default function SortDropdown({handleSort}){
+
+  //function determines which sort user clicked on and passes up that result to app.jsx which is where actual sorting logic will take place since movieList is there
+  function whichSort(event){
+    handleSort(event.target.value);
+  }
+
+  return(
+    <select onChange = {whichSort} id ="sort-dropdown">
+      <option value = "default" >Sort By</option>
```

Comment:
Let's keep as name "Default" here, and instead move the "sort by" text next to the selector